python-sdk/src/astro/utils/dataframe.py

```diff
@@ -17,3 +27,32 @@ def convert_columns_names_capitalization(
         elif columns_names_capitalization == "upper":
             df.columns = [col_label.upper() for col_label in df.columns]
     return df
+
+
+def convert_to_file(df: pd.DataFrame) -> File:
+    """
+    Passes a dataframe into a File using parquet as an efficient storage format. This allows us to use
+    Json as a storage method without filling the metadata database. the values for conn_id and bucket path can
+    be found in the airflow.cfg as follows:
+
+    [astro]
+    dataframe_storage_conn_id=...
+    dataframe_storage_url=///
+    :param df: Dataframe to convert to file
+    :return:
+    """
+    unique_id = random.choice(string.ascii_lowercase) + "".join(
+        random.choice(string.ascii_lowercase + string.digits) for _ in range(64)
+    )
+
+    # importing here to prevent circular imports
+    from astro.files import File
+
+    file = File(
+        path=DATAFRAME_STORAGE_URL + "/" + unique_id + ".parquet",
```

Comment:
@feluelle if you have two slashes it resolves to a single slash. 

foo//bar=foo////bar=foo/bar

Reply:
Okay, but this might in 0,00000001% not be what the user want. For example, in aws s3 you can have `/` as part of the prefix. You can also have `//` and it seems `File` wouldn't allow it. 😅

Reply:
@feluelle I think this is a case of "don't let the perfect be the enemy of the good". If we see this causing problem for users it can be fixed in a patch, but I believe this helps a lot more people than it hurts.